Makefile

```diff
@@ -322,6 +326,22 @@ regenerate-test-protos: build
 		${GENERATE_SRCS} --tfiws_out=FileNaming=DropPath:Tests/SwiftProtobufTests $$t; \
 	done
 
+#
+# Helper to update the .proto files copied from the google/protobufs distro.
+#
+update-proto-files:
```

Comment:
Would it be difficult to add an existence check for `${GOOGLE_PROTOBUFS_CHECKOUT}` and print a nice error message that says "Please clone google/protobuf as a sibling of swift-protobuf"?

(Or even clone it for them... but then you'd have to choose between http and ssh.)